scripts/oracle-setup.sh

```
@@ -0,0 +1,329 @@
+log_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+# Detect OS
+detect_os() {
+    if [ -f /etc/oracle-release ] || [ -f /etc/redhat-release ]; then
+        OS="oracle"
+        PKG_MANAGER="dnf"
+    elif [ -f /etc/debian_version ]; then
+        OS="ubuntu"
+        PKG_MANAGER="apt"
```

Shellcheck (0.11.0)

[warning] PKG_MANAGER appears unused. Verify use (or export if used externally).

(SC2034)